### ISSUE TITLE
WCAG 2.1: 1.4.13: Tooltip component is not keyboard-dismissable

### ISSUE THREAD

**Issue by dbjorge** [OfficeDev/office-ui-fabric-react#9435]
Migrated from https://github.com/microsoft/accessibility-insights-web/issues/777

## Codepen demonstrating the issue

https://codepen.io/dbjorge/pen/QXbvzN (this is just an export of the default Tooltip example from https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip as of writing)

## Actual behavior:

* Go to the codepen
* Click the "Hover Over Me" element
* Press "Tab" then "Shift+Tab" to simulate keyboard navigating to element
* Wait for tooltip to appear
* Press "Esc"
* Observe that tooltip does not disappear

## Expected behavior:

tooltip should be dismissable using esc key

## Documentation describing expected behavior

From https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

> Success Criterion 1.4.13 Content on Hover or Focus
> Where receiving and then removing pointer hover or keyboard focus triggers additional content to become visible and then hidden, the following are true:
> ...
> Dismissable
> A mechanism is available to dismiss the additional content without moving pointer hover or keyboard focus, unless the additional content communicates an input error or does not obscure or replace other content;

**Comment by natalieethell** [comment]
Hey @dbjorge thanks for submitting this issue and providing a CodePen for us! We're looking into it.
